satellite/satellitedb/dbx/satellitedb.dbx.go

```diff
@@ -18,8 +18,9 @@ import (
 
 	"github.com/lib/pq"
 
-	"github.com/mattn/go-sqlite3"
 	"math/rand"
+
+	"github.com/mattn/go-sqlite3"
```

Comment:
this should go with the other github.com imports

Reply:
This files auto-generated and it says I shouldn't edit it.